package.json

```diff
@@ -1,65 +1,71 @@
 {
-  "name": "virtualcoffee-io",
-  "version": "2.0.0",
-  "description": "The public site for virtualcoffee.io",
-  "scripts": {
-    "preinstall": "npx only-allow yarn",
-    "start": "npx grunt start",
-    "build": "ELEVENTY_ENV=production npx grunt release-prep",
-    "serve": "npx grunt connect:server",
-    "build-preview": "yarn build && yarn serve"
-  },
-  "repository": "git://github.com/Virtual-Coffee/virtualcoffee.io.git",
-  "author": "Virtual Coffee <hello@virtualcoffee.io> (https://virtualcoffee.io/)",
-  "license": "CC-BY-3.0",
-  "bugs": {
-    "url": "https://github.com/Virtual-Coffee/virtualcoffee.io/issues"
-  },
-  "homepage": "https://github.com/Virtual-Coffee/virtualcoffee.io#readme",
-  "devDependencies": {
-    "@11ty/eleventy": "^0.12.1",
-    "@11ty/eleventy-navigation": "^0.3.2",
-    "@11ty/eleventy-plugin-rss": "^1.1.1",
-    "@11ty/eleventy-plugin-syntaxhighlight": "^3.1.1",
-    "@fullhuman/postcss-purgecss": "^4.0.3",
-    "@lodder/grunt-postcss": "^3.0.1",
-    "autoprefixer": "^10.2.5",
-    "cssnano": "^5.0.1",
-    "grunt": "^1.3.0",
-    "grunt-cli": "^1.4.2",
-    "grunt-contrib-clean": "^2.0.0",
-    "grunt-contrib-concat": "^1.0.1",
-    "grunt-contrib-connect": "^3.0.0",
-    "grunt-contrib-copy": "^1.0.0",
-    "grunt-contrib-uglify": "^5.0.1",
-    "grunt-contrib-watch": "^1.1.0",
-    "grunt-filerev": "^2.3.1",
-    "grunt-filerev-assets": "^1.0.0",
-    "grunt-replace": "^2.0.1",
-    "grunt-sass": "^3.1.0",
-    "grunt-shell": "^3.0.1",
-    "load-grunt-tasks": "^5.1.0",
-    "lodash": "^4.17.21",
-    "luxon": "^1.26.0",
-    "markdown-it": "^12.0.5",
-    "markdown-it-anchor": "^8.3.0",
-    "postcss": "^8.2.10",
-    "prettier": "^2.2.1",
-    "sass": "^1.32.8",
-    "time-grunt": "^2.0.0"
-  },
-  "dependencies": {
-    "@11ty/eleventy-cache-assets": "^2.1.0",
-    "@imgix/js-core": "^3.1.3",
-    "bootstrap": "^4.6.0",
-    "dotenv": "^8.2.0",
-    "faker": "^5.5.3",
-    "graphql": "^15.5.0",
-    "graphql-request": "^3.4.0",
-    "markdown-it-attrs": "^4.0.0",
-    "markdown-it-toc-done-right": "^4.2.0",
-    "node-fetch": "^2.6.1",
-    "only-allow": "^1.0.0"
-  },
-  "main": "index.js"
+	"name": "virtualcoffee-io",
+	"version": "2.0.0",
+	"description": "The public site for virtualcoffee.io",
+	"scripts": {
+		"preinstall": "npx only-allow yarn",
+		"start": "npx grunt start",
+		"build": "ELEVENTY_ENV=production npx grunt release-prep",
+		"serve": "npx grunt connect:server",
+		"build-preview": "yarn build && yarn serve",
+		"prepare": "husky install"
```

Comment:
Let's add a `format` script here that runs prettier. Something like `"format": "npx prettier --write ."`

Reply:
Have it match the commands given to the `lint-staged` command

Reply:
Great idea. I'll get this done shortly.